Solutions/Infoblox NIOS/Data Connectors/Connector_Syslog_Infoblox.json

```diff
@@ -8,30 +8,30 @@
      { 
          "metricName": "Total data received", 
          "legend": "InfobloxNIOS", 
-         "baseQuery": "Infoblox_all" 
+         "baseQuery": "Infoblox_consolidated" 
```

Comment:
Question to @ms-sapangoel : Do we expect analysts to use **Infoblox_consolidated**? I would assume **Infoblox** would be a better choice for the primary parser.